crates/core/src/session.rs

```diff
@@ -38,6 +42,8 @@ pub struct NodeSession {
     pub address: SocketAddr,
     pub session: SessionId,
     pub last_seen: DateTime<Utc>,
+    pub forwarding_limiter:
+        Arc<RateLimiter<governor::state::NotKeyed, InMemoryState, DefaultClock>>,
```

Comment:
you are already importing `InMemoryState`, maybe also good to import `NotKeyed` for consistency?